content/en/docs/gke/deploy/management-setup.md

```diff
@@ -6,14 +6,26 @@ weight = 3
 
 This guide describes how to setup a management cluster which you will use to deploy one or more instances of Kubeflow.
 
+The management cluster is used to run [Cloud Config Connector](https://cloud.google.com/config-connector/docs/overview). Cloud Config Connector is a Kubernetes addon that allows you to manage Google Cloud resources through Kubernetes.
+
 While the management cluster can be deployed in the same project as your Kubeflow cluster, typically you will want to deploy
 it in a separate project used for administering one or more Kubeflow instances.
 
-The management cluster is used to run [Cloud Config Connector](https://cloud.google.com/config-connector/docs/overview).
-
 Optionally, the cluster can be configured with [Anthos Config Managmenet](https://cloud.google.com/anthos-config-management/docs) 
 to manage GCP infrastructure using GitOps.
 
+## FAQs
+
+* Where is `kfctl`?
+
+    `kfctl` is deprecated for Google Cloud, because its functionality has been replaced by generic tools including [Make](https://www.gnu.org/software/make/), [Kustomize](https://kustomize.io), [kpt](https://googlecontainertools.github.io/kpt/), and [Cloud Config Connector](https://cloud.google.com/config-connector/docs/overview).
```

Comment:
Suggestion: 

- The whole "don't make me think" logic:
```diff
- * Where is `kfctl`?
+ * Where is `kfctl`? It's deprecated

[explanation in more detail]
```

- Also, maybe some shortening:

```diff
-  `kfctl` is deprecated for Google Cloud, because its functionality has been replaced by generic 
+  `kfctl` is deprecated for Google Cloud—use generic 
...
```

- Maybe add a bit more information here, so the users don't have to click on Issue 123, to find out why kfctl is gone (as per https://github.com/kubeflow/gcp-blueprints/issues/123) - FAQ should help from the beginning, while referring to another link add work to the readers :

e.g. this big diff. between GCP's approach and other cloud providers w.r.t. kfctl: (cc @jlewi WDYT)

```diff
+ The decision is specific to Google Cloud, other platforms may continue to use kfctl.

[and one-two sentences from "Why did we stop using kfctl?" in issue 123] 
```

This will answer many users' concerts over why it's gone. Again, let's include this in the FAQ and not hide most of the good answers in Issue 123.

Reply:
@Bobgy  I might avoid saying "kfctl is deprecated" and might say something more narrow like kfctl is no longer being used to apply reourses There are other places where having a CLI can still be beneficial e.g.

* kfctl is still used for mirror'ing images
* kfctl is used with private GKE to create the DNS name
* kfctl can be used to run our kpt functions without using docker. Could be important for running containerized deployments

Or for example we might create a "diagnose me" tool baked into kfctl.

Reply:
Thanks @jlewi for the extra info, let me try to rephrase.

@8bitmp3 this is an early stage, we'll move kubeflow/gcp-blueprints#123 to documentation some time soon, so I'd prefer not spending too much time right now.